enterprise_gateway/mixins.py

```diff
@@ -431,6 +433,34 @@ def kernel_headers_default(self):
     def remote_hosts_default(self):
         return os.getenv(self.remote_hosts_env, self.remote_hosts_default_value).split(",")
 
+    # load_balancing_algorithm
+    load_balancing_algorithm_env = "EG_LOAD_BALANCE_ALGORITHM"
+    load_balancing_algorithm_default_value = "round-robin"
+    load_balancing_algorithm = Unicode(
+        load_balancing_algorithm_default_value,
+        config=True,
+        help="""Which load balancing algorithm does DistributedProcessProxy used.
+            Default round-robin,another use least-connection.
```

Comment:
Just some minor rephrasing...
```suggestion
        help="""Specifies which load balancing algorithm DistributedProcessProxy should use.
            Must be one of "round-robin" or "least-connection".
```
The default of "round-robin" will be specified in the output when running `jupyter enterprisegateway --help-all`